src/qibolab/result.py

```diff
@@ -70,16 +86,11 @@ def to_dict(self, average=True):
                             all the values for each shot are saved.
         """
         if average:
-            return {
-                "MSR[V]": self.msr.mean(),
-                "i[V]": self.i.mean(),
-                "q[V]": self.q.mean(),
-                "phase[rad]": self.phase.mean(),
-            }
-        else:
-            return {
-                "MSR[V]": self.msr.ravel(),
-                "i[V]": self.i.ravel(),
-                "q[V]": self.q.ravel(),
-                "phase[rad]": self.phase.ravel(),
-            }
+            self.compute_average()
+
+        return {
+            "MSR[V]": self.msr.ravel(),
```

Comment:
Is there a specific reason for raveling? You are loosing information, and it is still a Numpy array.

I have the feeling that this whole `to_dict` method is a service to a Dataframe constructor...

Reply:
I'm not against pandas: it is useful, but not deep inside qibolab. We should focus on rawer data, and have Dataframes in an external layer (if needed).

Reply:
> Is there a specific reason for raveling? You are loosing information, and it is still a Numpy array.

Not from my side. I was using `ravel` just because in a preliminary version that @stavros11 code for the `ExecutionResults` object he was using it. @stavros11 do you think that we can get rid of it?